tests/foreman/cli/test_repository.py

```diff
@@ -1595,7 +1597,7 @@ def test_positive_upload_content(self, repo):
 
         :CaseImportance: Critical
         """
-        ssh.upload_file(
+        default_sat.put(
             local_file=get_data_file(RPM_TO_UPLOAD), remote_file=f"/tmp/{RPM_TO_UPLOAD}"
```

Comment:
The kwargs need to be renamed, `put` and `get` use `local_path` and `remote_path`, not `local_file`/`remote_file`

Reply:
updated all instances in this file